tests/libtest/Makefile.inc

```diff
@@ -58,7 +58,7 @@ noinst_PROGRAMS = chkhostname libauthretry libntlmconnect                \
  lib1550 lib1551 lib1552 lib1553 lib1554 lib1555 lib1556 lib1557 \
  lib1558 lib1559 lib1560 lib1564 lib1565 \
  lib1591 lib1592 lib1593 lib1594 lib1596 \
- lib1900 lib1905 lib1906 lib1907 lib1908 \
+ lib1900 lib1905 lib1906 lib1907 lib1908 lib1910 \
```

Comment:
Why the `test1296` is not included? Maybe due to my lack of knowledge on the test suite :-/...

Reply:
It *is* included further up...

Reply:
Because the `libtest/` directory is for stand alone applications doing libcurl tests. `test1296` is a test file describing how to a specific test is done and in this case it uses the command line tool and isn't a libcurl-test so it shouldn't be included in this Makefile.